casts/splitNavigator/src/SplitNavigator/SplitBottomTabBar.tsx

```diff
@@ -104,6 +105,7 @@ function SplitBottomTabBarItem({
     keyProp: string;
     onPress: (key: string) => void;
 }) {
+    const handledTestID = `tab_bar_item_${keyProp.split('-')[0]}`
```

Comment:
Hm, I'm not sure that you can rely on `keyProp` here, as it might be dynamic 🤔

Reply:
Changed it in https://github.com/tonlabs/UIKit/pull/1427/commits/b3638f83c588eec1c43bd17742f301ec6a0bbd29